supertokens_python/recipe/jwt/api/implementation.py

```diff
@@ -25,4 +25,10 @@ async def jwks_get(
         self, api_options: APIOptions, user_context: Dict[str, Any]
     ) -> JWKSGetResponse:
         response = await api_options.recipe_implementation.get_jwks(user_context)
+
+        if response.validity_in_secs is not None:
+            api_options.response.set_header(
+                "Cache-Control", f"max-age={response.validity_in_secs}, must-revalidate"
+            )
+
         return JWKSGetResponse(response.keys)
```

Comment:
write an extra test wherein you query the jwks endpoint and check that resulting json. Make sure the resulting json only contains the keys props with the right values. No extra props should be there.

Reply:
Done